src/main/scala/BintrayPlugin.scala

```diff
@@ -0,0 +1,194 @@
+package bintray
+
+import bintry.Attr
+import sbt.{ AutoPlugin, Credentials, Global, Path, Resolver, Setting, Task, Tags, ThisBuild }
+import sbt.Def.{ Initialize, setting, task, taskDyn }
+import sbt.Keys._
+import sbt.Path.richFile
+
+object BintrayPlugin extends AutoPlugin {
+  import BintrayKeys._
+  import InternalBintrayKeys._
+
+  override def requires = sbt.plugins.JvmPlugin
+  override def trigger = allRequirements
+  
+  override def globalSettings: Seq[Setting[_]] = globalPublishSettings
```

Comment:
Yeah, sbt can have more than one build in an instance (e.g. when using ProjectRef)